ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -193,6 +193,12 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-openstack-ovn-etcd-scaling
+  optional: true
```

Comment:
i think we want to do an exercise of revisiting all our "optional" jobs to understand why they are optional and whether they fit one of the above reasons, or should be converted to blocking(or deleted).

So if you're telling me the plan is to gather data with an expectation of making it a blocking job in a month(or whatever shortish time period) i'm ok with it being optional for now.

But otherwise i'd say either don't add it at all (just gather data via periodics and then add it as a presubmit once you're happy w/ the periodic results) or add it as blocking(if you're already confident in it...sounds like not)

Reply:
It's hard to tell whether this job will break at some point if a regression is introduced into e.g. cluster-etcd-operator that would affect openstack jobs only.

I think you're right and in the current state of things, a periodic job would be enough:

* we'll use it to gather data and see how much it fails
* until (hasn't happen yet AFIK) we hit a regression somewhere that could have been spotted by having the presubmit jobs, we'll just rely on other platform's jobs for now.

Reply:
Our team uses number 3 for most of our repos and IMO are fairly good at gating on the jobs that we expect to pass. With a component like this, I think it's appropriate for this job to be optional since we have 5+ platforms that this repo effects. 

The core logic is always tested on AWS, but where there _could_ be interaction with cloud specific logic, we have the run if changed, but that's by no means perfect. It's then up to the reviewers to determine if they think it would actually warrant blocking on a specific platform.

This has worked well for us across the core parts of MAPI/CCM/CAPI so far, but it is definitely a team culture/discipline thing that we've developed, so I appreciate the concern.

TL;DR of this is that I think, given the repo, this is appropriate to be optional